client/plrdlg_common.cpp

```diff
@@ -283,14 +277,209 @@ static QString col_idle(const struct player *plr)
 }
 
 /**
-  Compares score of two players in players dialog
+  Compare scores of two players in players dialog
  */
 static int cmp_score(const struct player *player1,
                      const struct player *player2)
 {
   return player1->score.game - player2->score.game;
 }
 
+/************************************************************************/ /**
+    The name of the player's government
+  ****************************************************************************/
+QString col_government(const struct player *them)
+{
+  const struct player *me = client_player();
+
+  /*  'contact' gives the knowledge of other's government */
+  if (me == them || client_is_global_observer()
+      || player_has_embassy(me, them)
+      || player_diplstate_get(me, them)->contact_turns_left > 0) {
+    return government_name_for_player(them);
+  } else {
+    return _("?");
+  }
+}
+
+/************************************************************************/ /**
+    Compare culture of two players in players dialog,
+    needed to sort column
+  ****************************************************************************/
+static int cmp_culture(const struct player *player1,
+                       const struct player *player2)
+{
+  return player1->client.culture - player2->client.culture;
+}
+
+/************************************************************************/ /**
+    Show others player's culture to me if I am allowed to know it.
+  ****************************************************************************/
+QString get_culture_info(const struct player *them)
+{
+  const struct player *me = client_player();
+  bool gobs = client_is_global_observer();
+
+  // FIXME: Try to avoid this code duplication same code repeated in every
+  // column
+  if (them == nullptr || !them->is_alive) {
+    return _("-");
+  } else if (gobs
```

Comment:
Suggest to avoid using a variable only once (especially with such a nice name)
```suggestion

  // FIXME: Try to avoid this code duplication same code repeated in every
  // column
  if (them == nullptr || !them->is_alive) {
    return _("-");
  } else if (client_is_global_observer()
```